tariff/ngeso/shortrfc3339_test.go

```diff
@@ -0,0 +1,31 @@
+package ngeso
+
+import (
+	"testing"
+)
+
+var (
+	tTsBytes      = []byte("2023-04-20T14:30Z")
+	tTsStringRepr = "2023-04-20 14:30:00 +0000 UTC"
+)
+
+func TestMarshalling(t *testing.T) {
+	// Firstly, test that we can unmarshal into a struct.
+	ct := shortRFC3339Timestamp{}
+	err := ct.UnmarshalJSON(tTsBytes)
```

Comment:
```suggestion
    if err := ct.UnmarshalJSON(tTsBytes); err!=nil {
```

I've recently started using stretcherify/assert which would also give you

    assert.NoError(t, err)